packages/suite/src/components/suite/modals/ApplicationLog.tsx

```diff
@@ -89,8 +90,8 @@ export const Log = ({ onCancel }: Props) => {
                 />
                 <ActionColumn>
                     <Switch
-                        isChecked={!excludeBalanceRelated}
-                        onChange={actions.toggleExcludeBalanceRelated}
```

Comment:
toggleExcludeBalanceRelated is no more used, don't forget to remove it from line 36 https://github.com/trezor/trezor-suite/pull/5714/commits/9a5af0d87b8bd5830bba4cc38bd327d5ee60c0e5#diff-d2467ca75cefb9224e7d6dd3919ce7d164a25e6ee4fe5b976e543b617f52290dR36

Reply:
removed in next commit